api-model/pom.xml

```diff
@@ -1,14 +1,82 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <parent>
-        <artifactId>apicurio-registry-operator</artifactId>
-        <groupId>io.apicurio</groupId>
-        <version>2.2.1-SNAPSHOT</version>
-    </parent>
     <modelVersion>4.0.0</modelVersion>
 
+    <groupId>io.apicurio</groupId>
     <artifactId>apicurio-registry-operator-api-model</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
```

Comment:
I think the version of this should follow the version of the operator.  Is that 1.1.0?  @jsenko

Reply:
exactly, It's `1.1.0-dev` == `1.1.0-SNAPSHOT`

Reply:
OK great.  I'll let Ondrej know.  I picked the wrong version when I released this manually.

Reply:
Be aware that we have a time bomb waiting for us - when we try to release 2.2.0.Final of this in the future it will fail. :(

I don't know of any way to remove artifacts from maven central.